gk_cli/gk_cli/tokens.py

```diff
@@ -129,7 +129,7 @@ def _get_token_values(token_name):
         'but "spaghetti with meatballs, pasta alfredo" is not.\n'
     )
     response = prompt_user_with_help_check(token_values_prompt, help_message)
-    return tuple([value.encode('utf-8') for value in response.split(',')])
+    return tuple([value.strip().encode('utf-8') for value in response.split(',')])
```

Comment:
I would encourage *also* removing quote characters since they commonly cause issues in user-facing tools.